js/main.js

```diff
@@ -0,0 +1,34 @@
+var form = document.querySelector('#form');
+var imgSrc = document.querySelector('.photo-url');
+var placeholder = document.querySelector('#placeholder');
+
+// addEventListeners
+imgSrc.addEventListener('input', changeSrc);
+form.addEventListener('submit', handleSubmit);
+
+// functions for listener events
+function changeSrc(event) {
+  if (placeholder === '') {
+    imgSrc.setAttribute('src', '/images/placeholder-image-square.jpg');
+  } else {
+    placeholder.setAttribute('src', imgSrc.value);
+  }
+}
+
+function handleSubmit(event) {
+  event.preventDefault();
+  var title = document.querySelector('#title').value;
```

Comment:
This works! But I would recommend using the `elements` property on the form so that you don't have to query the DOM each time you submit the form.